comparative_benchmark/benchmark_lib.py

```diff
@@ -39,22 +39,13 @@ def _run(
     verbose: bool,
 ) -> utils.BenchmarkResult:
   model = benchmark.model
-  input_data = benchmark.input_data.artifacts[
-      def_types.ModelTestDataFormat.NUMPY_TENSORS]
-  expected_output = benchmark.expected_output.artifacts[
-      def_types.ModelTestDataFormat.NUMPY_TENSORS]
-
   data_type = model.model_parameters["data_type"]
   batch_size = model.model_parameters["batch_size"]
-  input_dims = input_data.data_parameters["tensor_dimensions"]
-  output_dims = expected_output.data_parameters["tensor_dimensions"]
   benchmark_definition = {
       "benchmark_name": benchmark.name,
       "framework": str(model.model_impl.framework_type),
       "data_type": data_type,
       "batch_size": batch_size,
-      "inputs": input_dims,
-      "outputs": output_dims,
```

Comment:
Since we don't define test data for each preprocessed input and output, these two fields are no longer available.

@beckerhe I'm not sure if we are using these two fields in the dashboard. If they are needed in the report, I can try to derive them from other places.

Reply:
We are not using them at the moment. So no objections.